moduleroot/Gemfile

```diff
@@ -82,8 +84,8 @@ group <%= group %> do
 <%            gem['length'] -%>
 <%          end.max -%>
 <% gems.each do |gem| -%>
-  gem '<%=
-  gem['gem'] %>', <%
+  gem "<%=
```

Comment:
This change of `'` to `"` is necessary so that interpolation of `minor_version ` is possible when the gemfile is generated.